capi/include/blazesym.h

```diff
@@ -353,6 +380,10 @@ typedef struct blaze_symbolize_inlined_fn {
    * Source code location information for the inlined function.
    */
   struct blaze_symbolize_code_info code_info;
+  /**
+   * Unused member available for future expansion.
+   */
+  uint8_t _reserved[8];
```

Comment:
do you support (plan to support) 32-bit build of blazesym? Because that will change pointer sizes in these structs and so we'd have to think about amount of reserved bytes needed

Reply:
I don't see a reason not to support it. Why does it matter, though? We reserve a bit more, percentage-wise, perhaps to something not an 8-byte boundary, but what impact does it have?

Reply:
I was just thinking if the idea is to reserve a space one extra pointer, then the reserved size might be smaller. But yes, in the gran scheme of things extra few bytes shouldn't be a big deal.

Reply:
The problem is that in general we can't anticipate what will be added. If we just add space for a `usize`/pointer then we run into problems if we were to extend the type by, say, an `Addr`/`u64`.